### ISSUE TITLE
Tensorflow text incompatible with Tensorflow 2.0.0-beta1

### ISSUE THREAD

**Issue by maxpoulain** [tensorflow/text#39]
Hi ! 

I just installed tensorflow==2.0.0-beta1 and I get the following issue :

```
ERROR: tensorflow-text 1.0.0b0 has requirement tensorflow==2.0.0b0, but you'll have tensorflow 2.0.0b1 which is incompatible.
```

How can I resolve this issue ? 

Thanks,

Maxime

**Comment by broken** [comment]
We will have a new release next week that will address this.

**Comment by broken** [comment]
We recently released 1.0.0-beta2 that is TF 2.0.0b1 compatible and should remain compatible for future releases of TF 2.0.
